validation/src/test/scala/hmda/validation/dsl/PredicateRegExSpec.scala

```diff
@@ -0,0 +1,108 @@
+package hmda.validation.dsl
+
+import hmda.parser.fi.ts.TsGenerators
+import org.scalatest.PropSpec
+import org.scalatest.prop.PropertyChecks
+import hmda.validation.dsl.PredicateRegEx._
+import hmda.validation.dsl.PredicateSyntax._
+import org.scalacheck.Gen
+
+class PredicateRegExSpec
+    extends PropSpec
+    with PropertyChecks
+    with TsGenerators {
+
+  // Email **************
+
+  property("A valid email must pass the email regex") {
+    forAll(emailGen) { email =>
+      email is validEmail
```

Comment:
The HMDA DSL doesn't work in tests. `email is validEmail` returns an _EditCheck_ `Success` or `Failure`, but that isn't the same as having a _spec_ expectation pass or fail. This line couldn't make the test fail, regardless of the input.

`validEmail.validate(email) mustBe true` would be a way to get the tests to reflect whether the validations pass or fail.